code/modules/virus2/effect/stage_3.dm

```diff
@@ -449,3 +449,24 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 		E.add_child(EE)
 		E.throw_at(get_step(src,pick(alldirs)), rand(1,4), rand(1,3))
 	..()
+
+
+/datum/disease2/effect/catvision
+	name = "Cattulism Syndrome"
+	stage = 3
+	max_count = 9
+	chance = 7
+	max_chance = 14
+	var/night_vision_strength = 0
+
+/datum/disease2/effect/catvision/activate(var/mob/living/carbon/mob)
+	night_vision_strength = mob.see_in_dark
+
+	if (mob.see_in_dark_override < 9)
+		spawn(rand(200,3000)
```

Comment:
Why are you spawning here, anyway?

Reply:
it was suggested to me that the night vision shouldn't come on instantly with the activation.

Reply:
Don't do that.